invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Home/Home.js

```diff
@@ -1,86 +1,24 @@
-import { FrontSiteRoutes } from '../../../routes/urls';
-import { goTo } from '../../../history';
-import { SearchBar } from '../../../common/components/SearchBar';
-import { document as documentApi } from '../../../common/api';
-import { BookGroup } from './components/BookGroup';
+import PropTypes from 'prop-types';
+import { Headline } from './Headline';
+import { SectionsWrapper } from './Sections';
 
 export class Home extends Component {
-  constructor(props) {
-    super(props);
-    this.state = { query: '' };
-  }
-
-  updateSearchQuery = (value, event) => {
-    this.setState({ query: event.target.value });
-  };
-
-  onSearchExecute = () => {
-    const query = encodeURIComponent(this.state.query);
-    goTo(FrontSiteRoutes.documentsListWithQuery(query));
-  };
-
   render() {
     return (
-      <Grid>
-        <Grid.Row centered columns={1}>
-          <Header size="huge">CERN Library</Header>
-          <SearchBar
-            currentQueryString={this.state.searchQuery}
-            onInputChange={this.updateSearchQuery}
-            executeSearch={this.onSearchExecute}
-            placeholder={'Search for books...'}
-          />
-        </Grid.Row>
-
-        <Grid.Row centered columns={1}>
-          <BookGroup
-            title={'Most Loaned Books'}
-            fetchDataMethod={documentApi.list}
-            fetchDataQuery={documentApi
-              .query()
-              .withDocumentType('BOOK')
-              .sortBy('-mostloaned')
-              .qs()}
-            viewAllUrl={FrontSiteRoutes.documentsListWithQuery(
-              '&sort=mostloaned&order=desc'
-            )}
-          />
-        </Grid.Row>
-
-        <Grid.Row centered columns={1}>
-          <BookGroup
-            title={'Most Recent Books'}
-            fetchDataMethod={documentApi.list}
-            fetchDataQuery={documentApi
-              .query()
-              .withDocumentType('BOOK')
-              .sortBy('mostrecent')
-              .qs()}
-            viewAllUrl={FrontSiteRoutes.documentsListWithQuery(
-              '&sort=mostrecent&order=desc'
-            )}
-          />
-        </Grid.Row>
-
-        <Grid.Row centered columns={1}>
-          <BookGroup
-            title={'Most Recent E-Books'}
-            fetchDataMethod={documentApi.list}
-            fetchDataQuery={documentApi
-              .query()
-              .withDocumentType('BOOK')
-              .withEitems()
-              .sortBy('-mostrecent')
-              .qs()}
-            viewAllUrl={FrontSiteRoutes.documentsListWithQuery(
-              'document_type:BOOK&sort=mostrecent&order=desc' +
-                '&aggr[0][has_eitems][value]=has_eitems.electronic versions'
-            )}
-          />
-        </Grid.Row>
-      </Grid>
+      <>
+        <Headline
+          renderElement={this.props.headline}
+          headlineImage={this.props.headlineImage}
+        />
+        <SectionsWrapper sections={this.props.sections} />
```

Comment:
I would use the same term we use for the style classes, `content` or `container` + Wrapper

Reply:
it has the class inside. I named it sections to highlight the possibility of overwriting the blocks of the landing page but if you think it is confusing, I can change that @zzacharo @ntarocco @equadon WDYT?